src/components/bottomPanel/tabs/IntegratedTerminal.vue

```diff
@@ -1,104 +1,45 @@
 <template>
-  <div class="relative h-full w-full bg-black">
-    <p v-if="errorMessage" class="p-4 text-center">{{ errorMessage }}</p>
-    <ProgressSpinner
-      v-else-if="loading"
-      class="absolute inset-0 flex justify-center items-center h-full z-10"
-    />
-    <div v-show="!loading" class="p-terminal rounded-none h-full w-full p-2">
-      <div class="h-full" ref="terminalEl"></div>
-    </div>
-  </div>
+  <Tabs value="0" class="h-full w-full">
```

Comment:
Can you register another bottom panel tab for the newly added interactive interminal and rename the old read-only terminal tab to `Logs`? The nested tabs does not seem to be right to me.